website/docs/en/config/experiments.mdx

```diff
@@ -168,26 +168,108 @@ In addition, you can also configure a `test` parameter for more fine-grained con
 The current lazy compilation aligns with the webpack implementation, **and is still in the experimental stage**. In some scenarios, lazy compilation might not work as expected, or the performance improvement may be insignificant.
 :::
 
+## experiments.lazyCompilation.backend
+
+- **Type:** `LazyCompilationBackendOptions`
+
+```ts
+type LazyCompilationBackendOptions = {
```

Comment:
We can change L114 from `backend?: { ... }` to `LazyCompilationBackendOptions` to reduce duplication of types in the document